exasol/bucketfs/__init__.py

```diff
@@ -0,0 +1,148 @@
+"""
+This module contains python api to programmatically access exasol bucketfs service(s).
+
+
+.. attention:
+
+    If no python api is required, one can also use CLI tools like CURL and HTTPIE to access bucketfs services.
+
+    Example's using CURL and HTTPIE
+    -------------------------------
+
+    #. Listing buckets of a bucketfs service
+
+        HTTPIE:
+          $ http GET http://127.0.0.1:6666/
+
+        CURL:
+          $ curl -i http://127.0.0.1:6666/
+
+
+    #. List all files in a bucket
+    
+        HTTPIE:
+          $  http --auth w:write --auth-type basic GET http://127.0.0.1:6666/default
+
+        CURL:
+          $ curl -i -u "w:write" http://127.0.0.1:6666/default
+"""
```

Comment:
maybe something for the user guide?

Reply:
I see this as rather relevant for the developer (alternative), a user of this lib is not the target audience for this in case of doubt I would rather locate it in the developer guide.